README.md

```diff
@@ -1841,6 +1842,7 @@ Options:
 -   `port` Port that Redis should be available on
 -   `parameter_group` The set of Redis parameters to use
 -   `num_nodes` Number of nodes in cache cluster
+-   `maintenance_window` specifies the weekly time range (of atleast 1 hour) in UTC during which maintenance on the cache cluster is performed. Default maintenance window is from sat:1:00-sat:2:00 EST or sat 05:00-06:00 UTC. 
```

Comment:
Very minor issue: UTC time is only 4 hours ahead of EST time during daylight savings, otherwise it's 5 hours ahead. Not sure if we care about the one hour difference tho.